Lesson14/Ayse/script.js

```diff
@@ -0,0 +1,282 @@
+// CODE HERE
+
+console.log('Example-15');
+
+points = 10;
+points += 5;
+console.log(points);
+
+// 16. Declare name="Alice", age=30, city="Paris". Log "Alice (30) lives in Paris" using template literals.
+// CODE HERE
+
+console.log('Example-16');
+
+const name16 = 'Alice';
+const age16 = 30;
+const city16 = 'Paris';
+console.log(`Hello! My name is ${name16}. I am ${age16} years old. I live in ${city16}.`);
+
+// 17. Declare variables x=5, y=10, z=15. Log their total sum.
+// CODE HERE
+
+console.log('Example-17');
+
+let x = 5;
+let y = 10;
+let z = 15;
+
+let totalSum = x + y + z;
+console.log(totalSum);
+
+// 18. Declare dividend=10 and divisor=3. Log the quotient (divisionResult) and difference (differenceResult).
+// CODE HERE
+
+console.log('Example-18');
+
+let divident = 10;
+let divisor = 3;
+let divisionResult = divident / divisor;
+let differenceResult = divident - divisor;
+console.log('Quotient: ',divisionResult, ' - ', 'Difference: ',differenceResult);
+
+// 19. Declare firstName and lastName. Create fullName by concatenating them with a space and log it.
+// CODE HERE
+
+console.log('Example-19');
+
+const firstNameEx19 = 'Ayse';
+const lastNameEx19= 'Luna';
+const fullNameEx19 = firstNameEx19 +' '+ lastNameEx19;
+console.log(fullNameEx19);
+
+// 20. Declare firstFactor=7 and secondFactor=2. Log the product.
+// CODE HERE
+
+console.log('Example-20');
+const firstFactor = 7;
+const secondFactor = 2;
+const product = firstFactor * secondFactor;
+console.log(product);
+
+
+// 21. Log the value of Math.PI.
+// CODE HERE
+
+console.log('Example-21');
+
+console.log(Math.PI);
+
+// 22. Declare counter=0. Increment it using three different methods (e.g., counter++, counter+=1, counter=counter+1) and log the result each time.
+// CODE HERE
+
+console.log('Example-22');
+
+let counterEx22 = 0;
+// First method: counter++
+counterEx22++;
+
+// Second method: counter+=1
+counterEx22 += 1 ;
+
+// Third method: counter = counter + 1
+counterEx22 = counterEx22+1;
+
+console.log(counterEx22);
+
+// 23. Declare initialTemperature=20. Increase it by 5 and log the result.
+// CODE HERE
+
+console.log('Example-23');
+
+let  initialTemperature = 20;
+initialTemperature -= 5;
+console.log(initialTemperature);
+
+// 24. Declare numberEx9=6. Increment it using the prefix ++ operator and log both the variable and the incremented value.
+// CODE HERE
+
+console.log('Example-24');
+
+let numberEx24 = 6;
+console.log(numberEx24);
+console.log(++numberEx24);
+
+// 25. Declare numberEx10=8. Increment it using the postfix ++ operator and log both the original variable and the incremented value.
+// CODE HERE
+
+console.log('Example-25');
+
+let numberEx25 = 8;
+console.log(numberEx25);
+console.log(numberEx25++);
+console.log(numberEx25);
+
+// 26. Declare numberEx11=-3. Increment it by 1 using prefix ++, then multiply the result by 2 and log the final value.
+// CODE HERE
+
+console.log('Example-26');
+
+let numberEx26 = -3;
+numberEx26 = ++numberEx26;
```

Comment:
When using prefix/postfix operators to increment a number, you don't need to re-assign it to itself. It's enough to do like this:
```suggestion
++numberEx26;
```